packages/teleport-project-generator-next/src/project-template.ts

```diff
@@ -20,9 +20,9 @@ const projectTemplate: GeneratedFolder = {
   "author": "TeleportHQ",
   "license": "MIT",
   "dependencies": {
-    "next": "^12.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "next": "^13.4.10",
```

Comment:
Are we sure we support this version?

Reply:
Yes, the current projects works as expected. And till now, we deployed only static sites. So, it's not going to break any projects. We need `next@13` in order to clear cache for multiple routes. `next@12` allows us only to clear a single route. 
But we generate `listing`, `details` pages for each content type. And we need to clear cache for the both, or else one of it will keep showing the data from the build time.

Reply:
I am going to downgrade again, found a workaround in next@12 itself. 13 has a lot of bugs with cache 👍